### PR TITLE
 do not return _unkown ids if collection was not found

### PR DESCRIPTION
**Pull request by hkernbach**
Throw instead of returning an `_unknown` collection ID if collection was not found.

### CODE REVIEW

**Comment by hkernbach** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/2126/

**Comment by hkernbach** [comment]
If review & jenkins is fine, also needs a PR for at least devel.

File: arangod/Utils/CollectionNameResolver.cpp [hunk 1]
```diff
@@ -310,7 +310,11 @@ std::string CollectionNameResolver::localNameLookup(TRI_voc_cid_t cid) const {
 
   // exactly as in the non-cluster case
   if (!ServerState::isDBServer(_serverRole)) {
-    return collection ? collection->name() : UNKNOWN;
```
Comment (graetzer) [inline arangod/Utils/CollectionNameResolver.cpp]: can you delete the string now ?

Comment (hkernbach) [reply arangod/Utils/CollectionNameResolver.cpp]: done.

**Comment by hkernbach** [comment]
@mchacki what are we going to do with this PR?

**Pull request closed without merging**